tests/sentry/deletions/test_project.py

```diff
@@ -134,7 +134,6 @@ def test_simple(self):
         assert not ProjectDebugFile.objects.filter(id=dif.id).exists()
         assert not File.objects.filter(id=file.id).exists()
         assert not ServiceHook.objects.filter(id=hook.id).exists()
-        assert not AlertRule.objects.filter(id=metric_alert_rule.id).exists()
```

Comment:
Why are we removing this?

Reply:
Nm, I think github's diff is being confusing

Reply:
yeah. it was removed and i had a bad conflict merge. fixing it plus it was causing a failed test